stream_alert/rule_processor/threat_intel.py

```diff
@@ -79,7 +78,8 @@ class StreamThreatIntel(object):
     # Class variable stores mapping between CEF normalized types and IOC types
     __normalized_ioc_types_mapping = {}
 
-    def __init__(self, table, region='us-east-1'):
+    def __init__(self, excluded_iocs, table, region='us-east-1'):
```

Comment:
since `excluded_iocs` could presumably be `None` or an empty dict `{}` (ie optional?) here, my preference would be to have this be the second arg (after `table`) or introduce `kwargs` to this initializer

Reply:
i think i might go with kwargs here -- it felt weird placing the excluded IOCs sandwiched between the two dynamo options, and I couldn't place it at the end because of the default value for region present in the args

Reply:
yep - I'd go with kwargs in this case too :). theoretically you could add the `region` as a kwarg here to since that is only necessary in some circumstance (for example - it's only really need for local testing, not when deployed in aws lambda)